### PR TITLE
feat: support multi-wallets

### PR DESCRIPTION
**Pull request by avimak**
1. generalizing wallet related access/docs (StarkNet is a multi-wallet ecosystem)
2. upgrading `@argent/get-starknet` to its latest version (which wraps [get-starknet](https://github.com/starknet-community-libs/get-starknet), StarkNet community's multi-wallet lib)
3. replaced faucet link to point towards a multi-wallet supporting TST faucet (argent locks their faucet to argent-only wallets)

### CODE REVIEW

**Comment by microbecode** [comment]
Thanks for the PR!
You are absolutely correct that this should reflect the multi-wallet ecosystem. When this readme was written there was just one wallet, and basically we just haven't updated the documentation since. Thanks for helping us with that.

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ All parameters need to be in integer format. If you need for example a _string_
 
 A typical use case would be for the multisig to contain some valuable assets. For testing this kind of realistic scenario you can use any StarkNet ERC-20 token.
 
-Probably easiest would be to use ArgentX's test token: https://argentlabs.github.io/argent-x/
```
[the remark this answers is not in the document]
Comment (microbecode) [reply README.md:84]: Sorry, only now read your comment about the argent-locked token. In that case I agree we should switch it. I'll give it a try!

[the remark this answers is not in the document]
Comment (microbecode) [reply README.md:84]: Works like a charm, and it's even the same token - thought it's some different token.

**Comment by avimak** [comment]
hey @microbecode , any updates on this PR?

**Comment by microbecode** [comment]
@avimak I left a review on one of the changes.

**Comment by avimak** [comment]
> @avimak I left a review on one of the changes.

sorry, can't find it, could u pls put here a link to it?

**Comment by microbecode** [comment]
> > @avimak I left a review on one of the changes.
> 
> sorry, can't find it, could u pls put here a link to it?

It's highlighted right on this PR page, a bit above.
But here's a link to the file in PR: https://github.com/eqlabs/starknet-multisig/pull/9/files/4a18d74673f93263d3461366e33aec3fb6239209#diff-da6498268e99511d9ba0df3c13e439d10556a812881c9d03955b2ef7c6c1c655

**Comment by avimak** [comment]
@microbecode - upgraded `starknet-react` lib to its latest version which support `get-starknet` **internally**

**Comment by microbecode** [comment]
> @microbecode - upgraded `starknet-react` lib to its latest version which support `get-starknet` **internally**

Thanks! I'm having a small holiday for the rest of the week. I'll get back to this next week.

**Comment by microbecode** [comment]
@avimak I now had a chance to test this out.
Does this code work for you? I can't even run it, since it's missing module get-starknet.
I was checking the documentation, and even that says you need to have get-starknet added explicitly: https://github.com/apibara/starknet-react#getting-started

**Pull request closed without merging**